std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
😞 IMO OOP was not the right design decision here.

Honestly at this point I'm inclined to say throw std.encoding in the trash and start over.